### PR TITLE
Use new website repo name

### PR DESCRIPTION
**Pull request by prince-chrismc**

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -84,8 +84,8 @@ jobs:
             mkdir -p pyrsia/usr/local/bin
             cp target/debug/pyrsia_node pyrsia/usr/local/bin
             cp target/debug/pyrsia pyrsia/usr/local/bin
-            chmod 755 pyrsia/usr/local/bin/pyrsia_node
-            chmod 755 pyrsia/usr/local/bin/pyrsia
+            chmod 777 pyrsia/usr/local/bin/pyrsia_node
+            chmod 777 pyrsia/usr/local/bin/pyrsia
```
Comment (mgrand) [inline .github/workflows/rust.yml:88]: I don't know the context of this, but usually making anything writable be anyone is a risky thing to do. Perhaps you should consider an alternative?

**Review by mgrand — APPROVED** [review]

File: .github/workflows/rust.yml (hunk 1 above)
Comment (sbtaylor15) [reply .github/workflows/rust.yml:88]: 755 was working fine in the demo.yml workflow.  We should keep 755.

Comment (prince-chrismc) [reply .github/workflows/rust.yml:88]: Sorry I did this on the side trying to fix https://github.com/pyrsia/pyrsia/runs/4742878764?check_suite_focus=true#step:7:51 but I am not sure it does

Excellent point, I was just trying to debug https://github.com/pyrsia/pyrsia/runs/4742813554?check_suite_focus=true#step:3:4

Seems to be two more bugs 😭

**Comment by prince-chrismc** [comment]
The problem was deeper than I expected marking this as draft

**Comment by sbtaylor15** [comment]
@prince-chrismc what is needed to wrap up this PR?

**Comment by prince-chrismc** [comment]
I havent looked into it. Main failed with the PR being merged so not everything is hunky dory 🤷

**Comment by prince-chrismc** [comment]
Mains magically passing 🤷

**Pull request closed without merging**